javascript/hangman.js

```diff
@@ -2,34 +2,79 @@ class Hangman {
   constructor(words) {
     this.words = words;
     // ... your code goes here
+    this.secretWord= this.pickWord(words)
+    this.letters=[];
+    this.guessedLetters="";
+    this.errorsLeft=10;
   }
 
   pickWord() {
     // ... your code goes here
+    //[hola,adios,perrro]
+    //multiplicamos el random por la longitud del arr y le quitamos los decimales
+    let random= Math.trunc(Math.random()*this.words.length)
+    //creamos una variable para pasarle la palabra random
+    let pickedWord =this.words[random]
+    //regresamos la palabra random
+    return pickedWord;
+
   }
 
   checkIfLetter(keyCode) {
-    // ... your code goes here
+    if(keyCode>=65 &&keyCode<=90){
```

Comment:
Esta expresion ya nos devuelve un booleno por lo que podemos prescindir de los condicionales
```suggestion
    return keyCode>=65 &&keyCode<=90
```